src/lib.rs

```diff
@@ -133,6 +133,9 @@ pub type ssize_t = isize;
 pub enum FILE {}
 pub enum fpos_t {} // TODO: fill this out with a struct
 
+#[cfg(not(any(dox, target_env = "msvc")))]
```

Comment:
Ah yeah let's split this between `src/unix` and `src/windows` if that's the case, and then yeah you'll want to add something in `doc.rs` so the travis builds can continue to build documentation for this